eng/tools/gen-sdk/vite.config.ts

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from "vite";
```

Comment:
Do we need to have different copies of this file and the tspconfig file as well?

Reply:
`tsconfig.json` is required as each tool has slight differences. I removed vite.config.ts and will consolidate it while adding tests in future PR.